packages/block-library/src/search/test/edit.native.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * External dependencies
+ */
+import renderer from 'react-test-renderer';
+
+/**
+ * WordPress dependencies
+ */
+import { Icon } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import SearchEdit from '../edit.native.js';
+
+const defaultAttributes = {
+	label: 'Search',
+	buttonText: 'Search Button',
+	buttonPosition: 'button-outside',
+	showLabel: true,
+};
+
+const getTestComponent = ( attributes = {} ) => {
+	const finalAttrs = { ...defaultAttributes, ...attributes };
+	return renderer.create(
+		<SearchEdit attributes={ finalAttrs } setAttributes={ jest.fn() } />
+	);
+};
+
+const getLabel = ( instance ) => {
+	return instance.findByProps( {
+		className: 'wp-block-search__label',
+	} );
+};
+
+const getButton = ( instance ) => {
+	return instance.findByProps( {
+		className: 'wp-block-search__button',
+	} );
+};
+
+const getSearchInput = ( instance ) => {
+	return instance.findByProps( {
+		className: 'wp-block-search__input',
+	} );
+};
+
+const hasComponent = ( instance, className ) => {
+	const components = instance.findAllByProps( {
+		className,
+	} );
+	return components.length !== 0;
+};
+
+describe( 'Search block options', () => {
+	it( 'renders block without crashing', () => {
+		const component = getTestComponent();
+		const rendered = component.toJSON();
+		expect( rendered ).toBeTruthy();
+	} );
+
+	it( 'renders default block configuration', () => {
```

Comment:
Minor comment but wondering if this `it` should be a `describe` call. That way the lower expect calls can be in their own `it` calls. I'm not sure if there is any specific convention in Gutenberg but I've usually try to minimize the number of expectations in a single test.

Reply:
Great idea and thanks for the tip! I made some changes in bf3894898336ab5984eeede32eec6daf60600f6a. Let me know what you think. Thanks!